go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -752,6 +782,23 @@ func NameAndTypeTransform(row sql.Row, rowOperationSchema sql.PrimaryKeySchema,
 	return row, nil
 }
 
+// detectAndConvertToBoolean determines whether a column is potentially a boolean and converts it accordingly.
+func detectAndConvertToBoolean(columnVal interface{}, columnType sql.Type) (bool, bool) {
+	switch columnType {
+	case sql.Boolean, sql.Int8, sql.MustCreateBitType(1): // TODO: noms bool wraps MustCreateBitType
```

Comment:
I think it makes more sense to use `columnType.Type()` here and the case statement should be `case querytypes.INT8, querytypes.BIT` (or whatever the actual variable names are)